src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -226,6 +236,22 @@ public override bool Execute ()
         }
         _fileWrites.Add(monoConfigPath);
 
+        if (ExtraFilesToDeploy != null)
+        {
+            foreach (ITaskItem2 item in ExtraFilesToDeploy!)
```

Comment:
Out of curiosity, why `ITaskItem2` over `ITaskItem`? It doesn't seem like any of the escape related properties or methods are being used.

Reply:
Err.. I was trying something else, and seem to have left this change in. I'll fix it.